models/endpoints.info.models.js

```diff
@@ -0,0 +1,5 @@
+const endpoints = require('../endpoints.json')
```

Comment:
you could probs just require it in the controller tbh

Reply:
thought that but didn't know if it was better to be consistent to structure used throughout